script/jenkins/package.sh

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env bash
 set -euox pipefail
 
- ./_beats/dev-tools/jenkins_release.sh
+# This controls the defaults used the Jenkins package job. They can be
+# overridden by setting them in the environement prior to running this script.
+export SNAPSHOT="${SNAPSHOT:-true}"
+export PLATFORMS="${PLATFORMS:-+linux/armv7 +linux/ppc64le +linux/s390x +linux/mips64}"
```

Comment:
This is a good find, why do we even bother to build for platforms we don't support (yet)?

Reply:
it is what the Makefile uses, we can remove all non-x86 packages

Reply:
Agreed, we should just unset this and let `packages.yml` define this instead.